src/Command/DebugCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright 2017 Hostnet B.V.
+ */
+namespace Hostnet\Bundle\AssetBundle\Command;
+
+use Hostnet\Component\Resolver\Bundler\Asset;
+use Hostnet\Component\Resolver\Bundler\EntryPoint;
+use Hostnet\Component\Resolver\Config\ConfigInterface;
+use Hostnet\Component\Resolver\File;
+use Hostnet\Component\Resolver\Import\ImportFinderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DebugCommand extends Command
```

Comment:
Shouldn't the bulk of this command be present in the `asset-lib`, so people not using Symfony would be able to to comparable analysis?

Ideally bundle is just glue.

Reply:
Nvm, read over the code again, can't see how a significant part of this could be re-used in the lib without adding `symfony/console` there.